README.md

```diff
@@ -18,17 +17,18 @@ But it's not enough because one video could become famous and overload the serve
 It's the reason why we need to use a P2P protocol to limit the server load.
 Thanks to [WebTorrent](https://github.com/feross/webtorrent), we can make P2P (thus BitTorrent) inside the web browser, as of today.
 
-## Why is that cool ?
+### Why is that cool ?
 Servers are run independently by different people and organizations. They can apply wildly different moderation policies, so you can find or make one that fits your taste perfectly.
 
 By watching a video, you help the hosting provider to broadcast it by becoming a broadcaster of the video yourself. Each instance doesn’t need much money to broadcast the videos of its users.
 
-## Demonstration
+Shipped Version: **1.2.1**
```

Comment:
Is not good to have shipped version on top?

Reply:
I can put the shipped version on top, but I though that more important than the shipped version, was the text about what is Peertube.

Let me know where you want it, and I put it.

Reply:
I don't have any problem with it, as it don't hinder the working of the package.

Reply:
Well, Neither do I, so if you want I moved the shipped version on top  or if you are OK to let it here, I let you resolve the conversation ;)